rise-playlist.html

```diff
@@ -176,8 +168,8 @@
           duration = parseInt(duration, 10);
 
           if (!isNaN(duration)) {
-            setTimeout(function() {
-              self._playNext(index, "pause");
+            this.debounce("play-next", function() {
+              this._playNext(index, "pause");
```

Comment:
@stulees I don't need to cancel this particular timer, just the loading one. And from what I understand, `debounce` makes it so that this timer will never be called more than once, which is a really nice feature that I'd like to leverage - just in case. I don't really get what advantage `async` gives me?